satellite/marketing/marketingweb/server.go

```diff
@@ -70,31 +102,74 @@ func NewServer(logger *zap.Logger, config Config, listener net.Listener) *Server
 	return s
 }
 
-// ServeHTTP handles index request
-func (s *Server) ServeHTTP(w http.ResponseWriter, req *http.Request) {
+// Serves index page and renders offer and credits tables
```

Comment:
https://github.com/storj/storj/wiki/Style#comments

Reply:
done

Reply:
This is still not following the comment pattern `// <function name> .....`

Reply:
This looks like an older commit. I just pushed up updates to the formatting of the comments again. If you pull down latest you should see they all follow the correct format now.

Reply:
I still don't see the new changes

Reply:
Did you pull down latest for `fh/create-new-offer` ?

Reply:
`// Serves index page and renders offer and credits tables` I'm seeing this on github